### ISSUE TITLE
Keyboard Color Contrast Violates WCAG Standards

### ISSUE THREAD

**Issue by nullpointerexceptionkek** [S0KPiseth/AlphaKeys#2]
![image](https://github.com/user-attachments/assets/54a7a04c-a67f-4b76-8629-8abdbbdbc797)

This keyboard color, purple and black, is tough for users to see, so it is recommended that they choose a different color.

**Comment by S0KPiseth** [comment]
Yeah, that color combo is kind of hard to see. The issue has been resolved in the latest commit [`67dce1f`](https://github.com/S0KPiseth/AlphaKeys/commit/67dce1f128ff39e97ed1f1639240509c104ff801), and the purple and black are just placeholder colors for now. I'll update them to something more user-friendly as I plan to implement both light and dark themes. Feel free to leave your color suggestions, and I'll definitely consider them.
